### PR TITLE
Feat/user settings routes

### PR DESCRIPTION
**Pull request by mellifluus**
plz do not merge until i squash everything after one commit

made multiple commits to make it easier to discern changes

### CODE REVIEW

**Comment by Alsje15** [comment]
> plz do not merge until i squash everything after one commit
> 
> made multiple commits to make it easier to discern changes

**Comment by Alsje15** [comment]
Didn't mean to close it :(
Anyway, my last merge was several unsquashed commits - when is this/is this not a problem?

File: src/routes/user/me.route.js
Comment (Alsje15) [inline src/routes/user/me.route.js:1]: Change username/email?

**Comment by mellifluus** [comment]
@Alsje15 only a problem in the backend, since multiple commits will trigger multiple releases

**Comment by Alsje15** [comment]
> @Alsje15 only a problem in the backend, since multiple commits will trigger multiple releases

@mellifluus well then that happened prev merge when we changes verify :| :| :|

**Comment by mellifluus** [comment]
> > @Alsje15 only a problem in the backend, since multiple commits will trigger multiple releases
> 
> @mellifluus well then that happened prev merge when we changes verify :| :| :|

nope, one commit there as far as i can see: https://github.com/thirst-alert/thirst-alert-be/pull/35/commits

**Pull request merged**